src/components/content/order/orderStatus/OrderSubmitStatusAlert.tsx

```diff
@@ -3,106 +3,253 @@
  * SPDX-FileCopyrightText: Huawei Inc.
  */
 
+import { UseMutationResult, UseQueryResult } from '@tanstack/react-query';
 import React, { useMemo } from 'react';
 import { useStopwatch } from 'react-timer-hook';
+import submitResultStyles from '../../../../styles/submit-result.module.css';
 import {
     ApiError,
-    DeployedServiceDetails,
+    DeployRequest,
     Response,
-    serviceDeploymentState,
+    ServiceOrder,
+    ServiceOrderStatusUpdate,
     ServiceProviderContactDetails,
+    taskStatus,
 } from '../../../../xpanse-api/generated';
+import { convertMapToDetailsList } from '../../../utils/convertMapToDetailsList';
 import { convertStringArrayToUnorderedList } from '../../../utils/generateUnorderedList';
 import { OperationType } from '../types/OperationType';
 import { OrderSubmitResult } from './OrderSubmitResult';
-import { ProcessingStatus } from './ProcessingStatus';
 
 function OrderSubmitStatusAlert({
     uuid,
-    isSubmitFailed,
-    deployedServiceDetails,
+    submitDeploymentRequest,
+    redeployFailedDeploymentQuery,
+    getSubmitLatestServiceOrderStatusQuery,
+    getReDeployLatestServiceOrderStatusQuery,
     serviceProviderContactDetails,
-    isPollingError,
+    retryRequest,
+    onClose,
+    optionType,
```

Comment:
we do not need this extra field. 
The deployment properties must be taken from the old service. The order status service will only return the status of the order. No information about the service.